src/Components/Comments.jsx

```diff
@@ -19,9 +19,9 @@ function Comments() {
                 setIsLoading(false)
          });}
          }, [article_id])
-
-         console.log(comments[0])
-         return isLoading ? (<p>Loading!</p>) : ( 
+        
+         console.log(comments)
+         return isLoading ? (<p>Loading!</p>) : comments === undefined ? (<p>No Comments for this Article </p>) : ( 
```

Comment:
you're on the right track, but it would be better to use comments.length === 0, because comments will never be undefined, as your usestate on line 11 makes it begin as an (empty) array.